Exiled.API/Features/Map.cs

```diff
@@ -248,8 +250,15 @@ public static void ResetLightsColor()
         /// Gets a random <see cref="Locker"/>.
         /// </summary>
         /// <returns><see cref="Locker"/> object.</returns>
+        [Obsolete("Use GetRandomExiledLocker() instead.")]
         public static Locker GetRandomLocker() => Lockers.GetRandomValue();
 
+        /// <summary>
+        /// Gets a random <see cref="Features.Lockers.Locker"/>.
+        /// </summary>
+        /// <returns><see cref="Features.Lockers.Locker"/> object.</returns>
+        public static Lockers.Locker GetRandomExiledLocker() => Features.Lockers.Locker.List.GetRandomValue();
```

Comment:
That an horrible name

Reply:
Any suggestions?

Reply:
> Any suggestions?

Make breaking change and just call it GetRandomLocker

Reply:
So can I make everywhere breaking changes? (Where it's not bad)

Reply:
Yeah